app/dashboard/views.py

```diff
@@ -300,6 +301,42 @@ def new_interest(request, bounty_id):
     })
 
 
+@require_POST
+def postComment(request):
```

Comment:
Lets use lowercase with underscores style here to match with PEP 8: [https://www.python.org/dev/peps/pep-0008/#naming-conventions](https://www.python.org/dev/peps/pep-0008/#naming-conventions). Also, since this function handles a POST request as evidenced by the decorator, maybe we should just call it `comment` or maybe `new_comment`. What do you think?